es6-const-let/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>es6-const-let</title>
+</head>
+<body>
+  <script>
+    const greetings = [
+      'hello!',
+      'konichiwa!',
+      'nihao',
+      'bonjour',
+      'gutten tag',
+      'ola'
+    ];
+
+    function logAllGreetings(greetingList) {
+      let i = 0;
+      while (i < greetingList.length) {
+        console.log('Current greeting:', greetingList[i]);
+        i++;
+      }
+    }
+    logAllGreetings(greetings);
+
+    function getRandomGreeting(greetingList) {
+      const randomIndex = Math.floor(Math.random() * greetingList.length);
+      let greeting = greetingList[randomIndex];
+      if (randomIndex === 1) {
+        greeting = greeting + ' is a japanese greeting.';
+      }
+      return greeting;
+    }
+    console.log('Random greeting:', getRandomGreeting(greetings));
+
+    function addToGreetings(newGreeting, greetingList) {
+      greetingList.push(newGreeting);
+    }
+    addToGreetings('namaste', greetings);
+    logAllGreetings(greetings);
+
+    console.log('This variable is currently undefined due to hoisting, please fix: ', variable);
+    var variable = 'meow';
```

Comment:
You missed a variable:

---

![image](https://user-images.githubusercontent.com/16143771/127087500-e13b9aa9-6f55-4099-99bb-ac79c85e9cec.png)